test/Crate/Test/DBAL/Functional/ConnectionTest.php

```diff
@@ -43,16 +44,18 @@ public function testBasicAuthConnection()
         $auth = ['crate', 'secret'];
         $params = array(
             'driverClass' => 'Crate\DBAL\Driver\PDOCrate\Driver',
+            // TODO: Could work by inheriting.
+            // 'wrapperClass' => 'Crate\DBAL\Driver\PDOCrate\PDOConnection',
```

Comment:
Could you please clarify this? Why would it be needed, why `could`? Sorry if this relates to my lack of current DBAL knowledge.